<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-24T19:06:21 IST" name="Suite" finished-at="2024-02-24T19:06:35 IST" duration-ms="14201">
    <groups>
    </groups>
    <test started-at="2024-02-24T19:06:21 IST" name="Test" finished-at="2024-02-24T19:06:35 IST" duration-ms="14201">
      <class name="extentReports.Tests.Tests">
        <test-method is-config="true" signature="setup()[pri:0, instance:extentReports.Tests.Tests@78d6692f]" started-at="2024-02-24T19:06:21 IST" name="setup" finished-at="2024-02-24T19:06:32 IST" duration-ms="11788" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="login()[pri:0, instance:extentReports.Tests.Tests@78d6692f]" started-at="2024-02-24T19:06:33 IST" name="login" finished-at="2024-02-24T19:06:34 IST" duration-ms="1381" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="addToCart()[pri:0, instance:extentReports.Tests.Tests@78d6692f]" started-at="2024-02-24T19:06:34 IST" name="addToCart" finished-at="2024-02-24T19:06:34 IST" depends-on-methods="extentReports.Tests.Tests.login" duration-ms="269" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[Intentional fail]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: Intentional fail
at junit.framework.Assert.fail(Assert.java:50)
at extentReports.PageObjects.HomePage.AddToCart(HomePage.java:36)
at extentReports.Tests.Tests.addToCart(Tests.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addToCart -->
      </class> <!-- extentReports.Tests.Tests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
